cumulus/polkadot-parachain/src/main.rs

```diff
@@ -42,6 +44,20 @@ impl CliConfigT for CliConfig {
 	fn copyright_start_year() -> u16 {
 		2017
 	}
+
+	type StatementStoreDefault = EnableStatementStoreByDefault;
+}
+
+/// Extra command line arguments for the node.
+#[derive(clap::Parser)]
+pub struct ExtraCliArgs {
```

Comment:
Don't see where this is used. Maybe it is a leftover?

Reply:
yes indeed, a leftover, removed now